ci/fireci/fireci/commands.py

```diff
@@ -72,7 +72,13 @@ def smoke_tests(app_build_variant, test_apps_dir):
 def api_information(auth_token, repo_name, issue_number):
   """Comments the api information on the pr"""
 
-  gradle.run('apiInformation')
```

Comment:
You can instead call
```python
gradle.run('apiInformation', '--continue', check=False)
```

Also note the continue flag, it will make sure apiInformation runs for all sdks instead of stopping upon first failure.